src/gol.test.js

```diff
@@ -0,0 +1,85 @@
+import next, { countLiveNeighbours } from './gol';
+
+test('function next exist', () => {
+  expect(typeof next).toBe('function');
+});
+
+test('next return array of array', () => {
+  const input = [[]];
+  const output = next(input);
+  expect(Array.isArray(output)).toBe(true);
+  expect(Array.isArray(output[0])).toBe(true);
+});
+
+test('one live cell will die', () => {
+  const input = [[1]];
+  const expectedOutput = [[0]];
+  const output = next(input);
+  expect(output).toEqual(expectedOutput);
+});
+
+test('Any live cell with fewer than two live neighbours dies, as if by underpopulation', () => {
```

Comment:
I like the self descriptive names of tests and also the fact that they are aligned with the original business requirements.